katello-installer/katello-installer-base.spec

```diff
@@ -46,32 +45,27 @@ A set of tools for installation of Katello and a the default Capsule.
 %{_sbindir}/capsule-certs-generate
 %{_sbindir}/katello-certs-check
 
-%package -n capsule-installer
-Summary:   Puppet-based installer for a Katello Capsule
+%package -n foreman-installer-capsule
```

Comment:
We should set this package to obsolete the capsule-installer to prevent  this from being used still on brownfield deployments. This may cause the config files to be renamed slightly which may impact the migration.